docs/advanced-topics/reproducing.md

```diff
@@ -7,86 +7,95 @@ permalink: /advanced-topics/reproducing/
+the fuzz target with your build and test system, all you have to do is run this command:
 ```bash
 $ ./fuzz_target_binary <testcase_path>
 ```
 
-If this is a timeout bug, add the **-timeout=25** argument.
-If this is an OOM bug, add the **-rss_limit_mb=2048** argument.
-Read more on how timeouts and OOMs are handed
-[here]({{ site.baseurl }}/faq/#how-do-you-handle-timeouts-and-ooms).
+For timeout bugs, add the `-timeout=25` argument. For OOM bugs, add the
+`-rss_limit_mb=2048` argument. Read more on [how timeouts and OOMs are
+handled]({{ site.baseurl }}/faq/#how-do-you-handle-timeouts-and-ooms).
 
 Depending on the nature of the bug, the fuzz target binary needs to be built
 with the appropriate [sanitizer](https://github.com/google/sanitizers)
-(e.g. if this is a buffer overflow, with
+(for example, if it's a buffer overflow, build with
 [AddressSanitizer](http://clang.llvm.org/docs/AddressSanitizer.html)).
 
-If you are not sure how to build the fuzzer using the project's build system,
-you may also use Docker
-([how?]({{ site.baseurl }}/getting-started/new-project-guide/#prerequisites),
-[why?]({{ site.baseurl }}/faq/#why-do-you-use-docker)) commands 
-to replicate the exact build steps used by OSS-Fuzz and then feed the reproducer
-input to the fuzz target.
+If you're not sure how to build the fuzzer using the project's build system,
+you can also use Docker commands to replicate the exact build steps used by
+OSS-Fuzz, then feed the reproducer input to the fuzz target ([how?]({{
+site.baseurl }}/getting-started/new-project-guide/#prerequisites), [why?]({{
+site.baseurl }}/faq/#why-do-you-use-docker)).
 
 ## Building using Docker
 
 ### Pull the latest Docker images
 
+Docker images get regularly updated with a newer version of build tools, build
+configurations, scripts, and other changes. In some cases, a particular issue
+can be reproduced only with a fresh image being used. Pull the latest images
+by running the following command:
+
 ```bash
 $ python infra/helper.py pull_images
 ```
 
-  Docker images get regularly updated with a newer version of build tools, build
-  configurations, scripts, and other changes. In some cases, a particular issue
-  can be reproduced only with a fresh image being used.
-
 ### Build the image and the fuzzers
 
+Run the following commands:
+
 ```bash
 $ python infra/helper.py build_image $PROJECT_NAME
 $ python infra/helper.py build_fuzzers --sanitizer <address/memory/undefined> \
     --architecture <x86_64/i386> $PROJECT_NAME
 ```
 
-The `architecture` argument is only necessary if you want to specify `i386` configuration.
+The `sanitizer` used in the report is the value in the
+**Sanitizer** column. It's one of the following:
+  * **address** for AddressSanitizer
+  * **memory** for MemorySanitizer
+  * **undefined** for UndefinedBehaviorSanitizer
+
+**Note**: The `architecture` argument is only necessary if you want to specify
+`i386` configuration.
 
 ## Reproducing bugs
```

Comment:
Should the topics from here on down also be nested under "Building with Docker"? I wasn't sure if they were part of the same procedure, but it seem like they are.

Reply:
Hmmm, I feel like these topics may make more sense here as it's really needed only when reproducing bugs, not like when the user wants to build the project in general case.

Reply:
OK, I'll leave as-is.